File: 0217_Contains_Duplicate/0217_Contains_Duplicate.ts
function containsDuplicate(nums: number[]): boolean {
    const numbers = new Set<number>();

    for (let i=0; i<nums.length; i++) {
        // Found Duplicate
        if (numbers.has(nums[i])) {
            return true;
        }
        numbers.add(nums[i]);
    }
    return false;
};

File: 0217_Contains_Duplicate/0217_Contains_Duplicate.swift
class Solution {
    func containsDuplicate(_ nums: [Int]) -> Bool {
        var numbers: Set<Int> = []

        for num: Int in nums {
            // Found Duplicate
            if numbers.contains(num) {
                return true
            }
            numbers.insert(num)
        }
        return false
    }
}